topdown/crypto_test.go

```diff
@@ -465,3 +466,184 @@ KcZjiyUsFLvdC5de1MeT1rJjQEsiZxH+QPR88tuByUVG000lpA==
 	})
 
 }
+
+func Test_getPrivateKeyFromPem(t *testing.T) {
```

Comment:
Thanks for this, we have a broad set of tests for this function now.

```suggestion
func Test_getPrivateKeysFromPEMData(t *testing.T) {
```